doc/source/cluster/aws-tips.rst

```diff
@@ -71,3 +71,202 @@ You should see something like
+      - us-west-2
+      - v1.247348.0b251302
+
+.. note::
+
+    Using Amazon CloudWatch will incur charges, please refer to `CloudWatch pricing <https://aws.amazon.com/cloudwatch/pricing/>`_ for details.
+
+Basic Setup:
+~~~~~~~~~~~~
+
+1. Create a minimal YAML cluster config named ``cloudwatch-basic.yaml`` with the following contents:
+
+.. code-block:: yaml
+
+    provider:
+        type: aws
+        region: us-west-2
+        availability_zone: us-west-2a
+        # Start by defining a `cloudwatch` section to enable CloudWatch integration with your Ray cluster.
+        cloudwatch:
+            agent:
+                # Path to Unified CloudWatch Agent config file
+                config: "cloudwatch/example-cloudwatch-agent-config.json"
+            dashboard:
+                # CloudWatch Dashboard name
+                name: "example-dashboard-name"
+                # Path to the CloudWatch Dashboard config file
+                config: "cloudwatch/example-cloudwatch-dashboard-config.json"
+
+    auth:
+        ssh_user: ubuntu
+
+    available_node_types:
+        ray.head.default:
+            node_config:
+            InstanceType: c5a.large
+            ImageId: ami-0d88d9cbe28fac870  # Unified CloudWatch agent pre-installed AMI, us-west-2
+            resources: {}
+        ray.worker.default:
+            node_config:
+                InstanceType: c5a.large
+                ImageId: ami-0d88d9cbe28fac870  # Unified CloudWatch agent pre-installed AMI, us-west-2
+                IamInstanceProfile:
+                    Name: ray-autoscaler-cloudwatch-v1
+            resources: {}
+            min_workers: 0
+
+2. Download example CloudWatch agent and dashboard configuration files:
+
+   First, create a ``cloudwatch`` directory in the same directory as ``cloudwatch-basic.yaml``.
+   Then, download the `CloudWatch agent config <https://github.com/ray-project/ray/blob/master/python/ray/autoscaler/aws/cloudwatch/example-cloudwatch-agent-config.json>`_ and `CloudWatch dashboard config <https://github.com/ray-project/ray/blob/master/python/ray/autoscaler/aws/cloudwatch/example-cloudwatch-dashboard-config.json>`_ to the cloudwatch directory.
+
+3. Run ``ray up cloudwatch-basic.yaml`` to start your Ray Cluster.
+
+.. note::
+    This will launch your Ray cluster in ``us-west-2`` by default. When launching a cluster for a different region, you'll need to change your YAML cluster launcher config file's ``region`` AND ``ImageId``.
+    See the ``Unified CloudWatch Agent Images table`` above for available AMIs by region.
+
+4. Check out your Ray cluster's logs, metrics, and dashboard in the `CloudWatch Console <https://console.aws.amazon.com/cloudwatch/>`_!
+
+Advanced Setup:
+~~~~~~~~~~~~~~~
+
+Refer to `example-cloudwatch.yaml <https://github.com/ray-project/ray/blob/master/python/ray/autoscaler/aws/example-cloudwatch.yaml>`_ for a complete example.
+
+1. Choose an AMI with the Unified CloudWatch Agent pre-installed.
```

Comment:
Is it possible to set up your own AMI with the agent/are there aws docs/an explanation on how that would be done?

Reply:
I think we could minimally provide a link to the CloudWatch agent installation docs (https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/install-CloudWatch-Agent-on-EC2-Instance.html) and the EC2 AMI creation docs (https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AMIs.html#creating-an-ami).

If we wanted to streamline the experience, we could also share a script that automates Ray CloudWatch Agent AMI creation. This actually used to be built into the AWS Cluster Launcher code for CloudWatch, but we eventually removed it since the install would sometimes take 30 minutes+ when run at EC2 instance launch time.

Reply:
Thanks Alex and Patrick, added instructions with link to AWS docs.